components/dashboard/src/projects/NewProject.tsx

```diff
@@ -525,17 +525,14 @@ function GitProviders(props: {
         });
     }
 
-    // for now we exclude GitHub Enterprise
-    const filteredProviders = () => props.authProviders.filter(p => p.host === "github.com" || p.host === "bitbucket.org" || p.authProviderType === "GitLab");
-
     return (
         <div className="mt-8 border rounded-t-xl border-gray-100 dark:border-gray-800 flex-col">
             <div className="p-6 p-b-0">
                 <div className="text-center text-gray-500">
                     Select a Git provider first and continue with your repositories.
                 </div>
                 <div className="mt-6 flex flex-col space-y-3 items-center pb-8">
-                    {filteredProviders().map(ap => {
```

Comment:
Good catch!

I've removed this filter from the UI, because I consider that:
- Your unsupported provider is visible in the UI, but it might potentially fail somewhere down the line because we haven't fully tested it

is a better UX than:
- Your unsupported provider is not visible in the UI, because we haven't tested it yet

In other words, maybe it's better to try and give users more agency here. If things work out of the box, great! If they break in some specific way, we can file an issue and try to solve it quickly. (But if we completely hide these, we're sort of ensuring that they'll never work.)

Reply:
> is a better UX than:

really? 

how is a (untested ===) broken state better than hiding untested paths? 

> But if we completely hide these, we're sort of ensuring that they'll never work.

We got a lot of requests and scheduled several tasks to extend the support in this area in the past year, and this PR is an example for it. So this isn't true.

Reply:
I'm still biased towards leaving all doors open and all panels unlocked, so that savvy users can help us test and find their way around / fix their own problems, as opposed to shutting every door and having just one supported path.

However, I also understand your point of view, so I'll re-instate the UI filter I removed (and adjust it to show GitHub Enterprise as well).